onnxruntime/test/framework/inference_session_test.cc

```diff
@@ -1481,9 +1478,7 @@ TEST(InferenceSessionTests, Test3LayerNestedSubgraph) {
   InferenceSession session_object{so, GetEnvironment()};
 
 #ifdef USE_CUDA
-  CUDAExecutionProviderInfo epi;
-  epi.device_id = 0;
-  EXPECT_TRUE(session_object.RegisterExecutionProvider(std::make_unique<CUDAExecutionProvider>(epi)).IsOK());
+  EXPECT_TRUE(session_object.RegisterExecutionProvider(DefaultCudaExecutionProvider()).IsOK());
```

Comment:
It is good. But if you help us change it to 
```c++
  ASSERT_STATUS_OK(session_object.RegisterExecutionProvider(DefaultCudaExecutionProvider());
```
It will be even better.

Reply:
It is even better now